index.js

```diff
@@ -1,5 +1,8 @@
 module.exports = (nextConfig = {}) => {
   return Object.assign({}, nextConfig, {
+    publicRuntimeConfig: Object.assign({}, nextConfig.publicRuntimeConfig, {
+      nextImagesAssetPrefix: nextConfig.assetPrefix
+    }),
```

Comment:
Can't we move it to line 8? There we are initializing all configuration options.

Reply:
Unfortunately we can't move it from here because `webpack` part of next config is only used during build time. If you are in production "serve mode" aka `next start` webpack method wouldn't be called so root `publicRuntimeConfig` would not be touched.